set.mm

```diff
@@ -505121,6 +505183,56 @@ its graph has a given second element (that is, function value).
       UNRJUNABCTAULSUNRDNAUNAGUMGUAUBAUCUDAUMABCUEUKUFUGUHUIUJ $.
   $}
 
+  ${
+    quad3d.1 $e |- ( ph -> X e. CC ) $.
+    quad3d.2 $e |- ( ph -> A e. CC ) $.
+    quad3d.3 $e |- ( ph -> A =/= 0 ) $.
+    quad3d.4 $e |- ( ph -> B e. CC ) $.
+    quad3d.5 $e |- ( ph -> C e. CC ) $.
+    quad3d.6 $e |- ( ph -> ( ( A x. ( X ^ 2 ) ) + ( ( B x. X ) + C ) ) = 0 ) $.
+    $( Variant of quadratic equation with discriminant expanded.  (Contributed
+       by Filip Cernatescu, 19-Oct-2019.)  Deduction version.  (Proof Modified
```

Comment:
Proof Modified by -> Revised by

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4926/commits/7a68c47ad0d3d7310d3af9b732063ce7ae518b01